examples/AzureSDKDemoSwift/Source/BlobDownloadViewController.swift

```diff
@@ -157,8 +157,7 @@ extension BlobDownloadViewController: UITableViewDelegate, UITableViewDataSource
 
         // Match any blobs to existing transfers.
         // Update background color and progress.
-        if let transfer = blobClient.transfers.downloadedFrom(container: AppConstants.videoContainer, blob: blobName)
-            .first {
+        if let transfer = blobClient.downloads.firstWith(blobName: blobName) {
```

Comment:
This would be one of those situations where `firstWith` could easily lead to surprising results since, within a storage account, a blob name is unique only in combination with its container.

Reply:
It's true, I'm still not sure we're striking the right balance between user-friendliness (making it easy for the developer to find what they're looking for) and correctness (ensuring users can't introduce bugs because their assumptions are not entirely valid).